arangod/IResearch/IResearchAqlAnalyzer.h

```diff
@@ -66,6 +83,12 @@ class AqlAnalyzer final : public irs::analysis::analyzer{
 
     /// @brief memory limit for query.  1Mb by default. Could be increased to 32Mb
     uint32_t memoryLimit{ 1048576U };
+
+    /// @brief target type to convert query output. Could be
+    ///        string, bool, number. If missing - backward compatible mode is used
+    ///        strings are passed verbatim, nulls discarder/replaced other types
+    ///        silently discarded.
+    AnalyzerValueType returnType{AnalyzerValueType::Undefined};
```

Comment:
string by default?

Reply:
If we decide to drop old behaviour (see comment above) than yes.